config/.env.test

```
@@ -23,3 +23,4 @@ BHASINI_USER_ID=this_is_not_the_secret
 BHASINI_ULCA_API_KEY=this_is_not_the_secret
 GOOGLE_MAPS_API_KEY=this_is_not_the_secret
 READ_REPLICA_DATABASE_URL=ecto://postgres:postgres@127.0.0.1:5432/glific_test
+GUPSHUP_PARTNER_CLIENT_SECRET=this_is_not_the_secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GUPSHUP_PARTNER_CLIENT_SECRET key should go before the HTTP_PORT key

(UnorderedKey)